composer.json

```diff
@@ -14,13 +14,18 @@
         "symfony/framework-bundle": "~2.7|~3.1",
         "symfony/finder": "~2.7|~3.1",
         "symfony/console": "~2.7|~3.1",
-        "symfony/intl": "~2.7|~3.1"
+        "symfony/intl": "~2.7|~3.1",
+        "symfony/asset": "~2.7|~3.1",
+        "symfony/templating": "~2.7|~3.1",
+        "symfony/translation":  "~2.7|~3.1",
+        "doctrine/annotations": "~1.0"
     },
     "require-dev": {
         "symfony/yaml": "~2.7|~3.1",
         "symfony/browser-kit": "~2.7|~3.1",
         "symfony/twig-bundle": "~2.7|~3.1",
-        "symfony/phpunit-bridge": "~2.7|~3.1"
+        "symfony/phpunit-bridge": "~2.7|~3.1",
+        "phpunit/phpunit": "^4.8"
```

Comment:
allowing only 4.8 is a bad idea, as PHPunit 4.8 does not officially support PHP 7 (and will break totally on PHP 7.1 when mocking some classes). Maintained versions of PHPUnit (i.e. 5.x) must be allowed by the constraint.